docroot/themes/humsci/humsci_basic/readme.md

```diff
@@ -0,0 +1,54 @@
+# Stanford Basic theme
+
+Stanford Basic is used a base for all future H&S themes. It integrates selectively with Decanter Sass, styles and 
+patterns.
+
+## Requirements
+
+- Node 10.15+
+
+## Getting Started
+
+This theme contains it's own node module dependencies and build system separate from the root project. All commands 
+should be run from this theme's directory.
+
+- `npm install` - Install all node dependencies
+
+## Builds 
+
+Frontend assets are built using the Grunt task runner, but using npm scripts as shortcuts. Assets are compiled to a 
+`dist/` directory with a stylesheet generated for each sub-theme.
+
+- `npm run build:sass` - Compile Sass for production
+- `npm run watch` - Compile a build and watch for changes
+
+## Testing
+
+- `npm test` - Run linting and Sass tests
```

Comment:
we also need to figure out how to run and make our own behat tests, and unit tests for modules